src/odemis/gui/comp/legend.py

```diff
@@ -58,10 +58,15 @@ def __init__(self, parent, wid=-1, pos=(0, 0), size=wx.DefaultSize,
         self._stream_to_icon = (
             (stream.ARStream, img.getBitmap("icon/ico_blending_ang.png")),
             (stream.SpectrumStream, img.getBitmap("icon/ico_blending_spec.png")),
-            (stream.OpticalStream, img.getBitmap("icon/ico_blending_opt.png")),
             (stream.CLStream, img.getBitmap("icon/ico_blending_opt.png")),  # same as optical
             (stream.EMStream, img.getBitmap("icon/ico_blending_sem.png")),
             (stream.RGBStream, img.getBitmap("icon/ico_blending_goal.png")),
+            (stream.RGBUpdatableStream, img.getBitmap("icon/ico_blending_map.png")),
+            (stream.RGBCameraStream, img.getBitmap("icon/ico_blending_navcam.png")),
+            (stream.BrightfieldStream, img.getBitmap("icon/ico_blending_navcam.png")),
+            (stream.Static2DStream, img.getBitmap("icon/ico_blending_history.png")),
```

Comment:
Static2DStream includes many streams, including StaticFluoStream, which should be shown with optical icon.